src/Plugin.php

```diff
@@ -25,4 +25,20 @@ public function getCapabilities()
             'Composer\Plugin\Capability\CommandProvider' => 'Bamarni\Composer\Bin\CommandProvider',
         );
     }
+
```

Comment:
Spaces should be used instead of tabs

Reply:
Oof, sorry I didn't realize this. I force-pushed a commit with spaces now. Thank you.